src/CTA.Rules.Config/Utils.cs

```diff
@@ -109,14 +109,14 @@ public static string DownloadFile(string fileUrl, string destinationFile, int re
                         File.WriteAllBytes(destinationFile, fileContents);
                         break;
                     }
-                    catch (Exception ex)
+                    catch (Exception)
                     {
                         retryAttempts++;
                         Thread.Sleep(Constants.DefaultThreadSleepTime);
 
                         if (retryAttempts == retryCount)
                         {
-                            throw ex;
+                            throw;
```

Comment:
Why remove? shouldn't we throw the original exception so that the function using it has data around why this happened?

Reply:
This will still throw the original exception. The point of this is to maintain the stack trace of the original exception.

I originally thought this had come up as a build warning, which is why I changed it. But looking back, it might have just been a code quality warning. https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2200

By just calling "throw" instead of "throw ex" we preserver the stack trace of the error. So we would be able to see exactly where the real error was thrown. If we call "throw ex" the stack trace will only point us to this line here, which I assume won't be as helpful.